blog/se1/index.qmd

```diff
@@ -0,0 +1,28 @@
+---
+author: Caleb Kendra, Alish Chhetri, and Evan Nelson
+title: Programming Over Time
+date: '2023-11-29'
+date-format: YYYY
+categories: [post, developer development, software engineering]
+description: <em>This is our first blog about the `Software Engineering at Google` book!</em>
```

Comment:
You should add a more description statement that tells something about what you learned from reading the article.